src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+@use "../utils/variable" as *;
```

Comment:
Can't you just import it?

Reply:
![image](https://user-images.githubusercontent.com/12642122/222455922-775b97fb-d810-4089-bfce-e79760010cbc.png)
https://www.youtube.com/watch?v=EvCb0qg9ShI 2:25